### PR TITLE
Add parallels15 cask

### PR DESCRIPTION
**Pull request by andreineculau**
Identical to https://raw.githubusercontent.com/Homebrew/homebrew-cask/bfabf13824108181878ada89125eca82ea75d242/Casks/parallels.rb
before it got upgraded to v16 in https://github.com/Homebrew/homebrew-cask/pull/87490

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask-versions/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [ ] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask-versions/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/parallels15.rb [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+  url "https://download.parallels.com/desktop/v#{version.major}/#{version}/ParallelsDesktop-#{version}.dmg"
+  appcast "https://kb.parallels.com/eu/124724"
+  name "Parallels Desktop"
+  desc "Parallels Desktop is a desktop virtualization software"
+  homepage "https://www.parallels.com/products/desktop/"
+
+  auto_updates true
+  depends_on macos: ">= :sierra"
+  # This .dmg cannot be extracted normally
```
Comment (reitermarkus) [inline Casks/parallels15.rb:13]: Is this still the case actually?

File: Casks/parallels15.rb [hunk 2]
```diff
@@ -0,0 +1,62 @@
+cask "parallels15" do
+  version "15.1.4-47270"
+  sha256 "7a0a876c5a357c0744626117c359b09e28920b35ec9b63f2dbbafe3bd7a639fd"
+
+  url "https://download.parallels.com/desktop/v#{version.major}/#{version}/ParallelsDesktop-#{version}.dmg"
+  appcast "https://kb.parallels.com/eu/124724"
+  name "Parallels Desktop"
+  desc "Parallels Desktop is a desktop virtualization software"
```
Comment (reitermarkus) [inline Casks/parallels15.rb:8]: Remove the name from the description.

File: Casks/parallels15.rb (hunk 1 above)
Comment (andreineculau) [reply Casks/parallels15.rb:13]: Like I wrote, this is a paste of https://raw.githubusercontent.com/Homebrew/homebrew-cask/bfabf13824108181878ada89125eca82ea75d242/Casks/parallels.rb .
I cannot test.

Comment (reitermarkus) [reply Casks/parallels15.rb:13]: Please try removing it and the workaround in `preflight`, we'll see if CI passes.

**Review by reitermarkus — CHANGES_REQUESTED** [review]

File: Casks/parallels15.rb [hunk 3]
```diff
@@ -0,0 +1,62 @@
+cask "parallels15" do
+  version "15.1.4-47270"
+  sha256 "7a0a876c5a357c0744626117c359b09e28920b35ec9b63f2dbbafe3bd7a639fd"
+
+  url "https://download.parallels.com/desktop/v#{version.major}/#{version}/ParallelsDesktop-#{version}.dmg"
+  appcast "https://kb.parallels.com/eu/124724"
+  name "Parallels Desktop"
+  desc "A desktop virtualization software"
```
[the remark this answers is not in the document]
Comment (andreineculau) [reply Casks/parallels15.rb:8]: Which error? This one https://github.com/Homebrew/homebrew-cask-versions/commit/7757747d458e19bd4ec0c3013e00e61746c6c348 ?

**Comment by andreineculau** [comment]
@reitermarkus would you be so nice to explain your reasoning in trying without the workaround since both 14 (in this repo) and 16 (current) have the workaround?

By now, 15 has been installed with the workaround numerous times. Now we try without it and hope both that the CI is green AND that the CI would actually catch a regression (maybe it will be green but the installation still doesn't work for some reason, but there's no test for the corner case).

The way that i perceived the versions cask tap is simply an iterative "brew extract foo@<n-1>" tap (where n is the latest version), no need for patching unless newer brew has backward incompatibilities or a formula starts breaking.

**Comment by reitermarkus** [comment]
The workaround was added for version 14, so it may not be necessary for 15 and simply still be present because nobody tried removing it until now. So now that I noticed it, we should try removing it if possible.

**Review by reitermarkus — DISMISSED** [review]

**Comment by reitermarkus** [comment]
> maybe it will be green but the installation still doesn't work for some reason

More often than not CI is actually failing when there is no failure locally. In any case, if there really is an issue, we can re-add the workaround if someone reports it.

**Review by reitermarkus — APPROVED** [review]
